src/mson.js

```diff
@@ -204,7 +206,12 @@ function handle(name, element, { parent = null, spaces = 4, marker = '+',
  * Render out a piece of MSON from refract element instances.
  */
 export function renderDataStructure(dataStructure) {
-  const mson = dataStructure.content[0];
```

Comment:
In Drafter there was a bug that data structure content was incorrectly wrapped in an array and seems this was assumed as behaviour here. Since the Swagger adapter doesn't have this bug I'd made it conditional below. In future we can remove this check.